src/index.html

```diff
@@ -16,7 +37,389 @@
+                  />
+                </a>
+              </div>
+              <h3 class="details__title">Range & Integrated lights</h3>
+              <p class="details__description">
+                The removable battery has up to 70km battery autonomy and weighs
+                only 2.4 kg. Lights integrated into the frame give you always-on
+                visibility day and night.
+              </p>
+            </article>
+
+            <article class="details__item">
+              <div class="details__images">
+                <a
+                  href="#"
+                  class="details__link--square"
+                >
+                  <img
+                    src="./images/hydraulic-disc-brakes-1.png"
+                    alt="Hydraulic disc brakes & Lightweight example"
+                    class="details__image"
+                  />
+                </a>
+                <a
+                  href="#"
+                  class="details__link--wide"
+                >
+                  <img
+                    src="./images/hydraulic-disc-brakes-2.png"
+                    alt="Hydraulic disc brakes & Lightweight example"
+                    class="details__image"
+                  />
+                </a>
+              </div>
+              <h3 class="details__title">
+                Hydraulic disc brakes & Lightweight
+              </h3>
+              <p class="details__description">
+                Brakes with total stopping power the second you make contact.
+                The removable battery has up to 70km battery autonomy and weighs
+                only 2.4 kg. Lights integrated into the frame give you always-on
+                visibility day and night.
+              </p>
+            </article>
+            <a
+              href="#"
+              class="details__button button"
+            >
+              Explore
+            </a>
+          </div>
+        </section>
+        <section
+          class="page__section contacts"
+          id="contacts"
+        >
+          <h2 class="page__section-title contacts__title">Contact us</h2>
+
+          <div class="contacts__content">
+            <form
+              onsubmit="this.reset(); return false;"
+              class="contacts__form-field"
+            >
+              <input
+                type="text"
+                placeholder="Name: John Dee"
+                class="contacts__input"
+                required
+              />
+              <input
+                type="text"
+                placeholder="Email: example@gmail.com"
+                class="contacts__input"
+                required
+              />
+              <textarea
+                name="message"
+                placeholder="Message:"
+                rows="4"
+                class="contacts__input contacts__input--textarea"
+                required
+              ></textarea>
+              <button class="contacts__button button">Send</button>
```

Comment:
It makes more sense using submit input, if you are using the form like it is, without any additional JS processing
```suggestion
              <input type="submit" value="Send"  class="contacts__button button"/>
```